core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/ProcessUtil.java

```diff
@@ -42,7 +44,7 @@
    * @param stderr a consumer where stderr will be redirected
    * @throws IOException
    */
-  public static void process(Process p, LineConsumer stdout, LineConsumer stderr)
+  public static void readOutput(Process p, LineConsumer stdout, LineConsumer stderr)
```

Comment:
-1 to change method name. It can be used by someone else.

Reply:
Agree